test/smoke/src/sql/areas/notebook/createBook.test.ts

```diff
@@ -61,6 +61,9 @@ export function setup() {
 					// Try our best to clean up but don't fail the test if we can't
 				}
 			}
+			const app = this.app as Application;
+			// Workaround for error notification trying to read deleted books
```

Comment:
I was hitting issues with the toasts from these tests blocking the dialog. It's something I plan on coming back to fix in a better way but hopefully this should at least prevent issues in the near term.